### PR TITLE
refactor: command dependence use spf13/cobra instead

### PR DESCRIPTION
**Pull request by MasterKenway**
**What this PR does**:

refactor command dependence

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by cityiron — APPROVED** [review]

File: cmd/pixiu/control.go [hunk 1]
```diff
@@ -41,99 +45,108 @@ var (
...
+	// CURRENTLY USELESS
+	logFormat string
+
+	limitCpus string
 
-			err := logger.InitLog(logConfPath)
+	// Currently default set to false, wait for up coming support
+	initFromRemote = false
+
+	startCmd = &cobra.Command{
```
Comment (AlexStocks) [inline cmd/pixiu/control.go:61]: 不要用 startCmd，改成 startGatewayCmd，启动命令改成 --start-gateway，为将来 sidecar 模式留下空间

Comment (MasterKenway) [reply cmd/pixiu/control.go]: resolved

**Review by AlexStocks — APPROVED** [review]
LGTM

**Comment by MasterKenway** [comment]
readme needed to be updated

**Review by ztelur — APPROVED** [review]

**Pull request merged**